pandas/core/internals.py

```diff
@@ -3597,7 +3597,14 @@ def value_getitem(placement):
                 return value
         else:
             if value.ndim == self.ndim - 1:
-                value = value.reshape((1,) + value.shape)
+                # gh-13012: 'reshape' is deprecated for Categorical
+                # and Series and will be removed in a subsequent version,
+                # hence the two checks done before calling 'reshape'
+                if isinstance(value, ABCSeries):
+                    value = value._values
```

Comment:
oh I see.

ok, need an internal function to do this then (rather than having this duplicated code)

put in ``pandas.types.concat`` maybe ``compat_reshape``